org-code-javabuilder/protocol/src/main/java/org/code/protocol/StatusMessageKey.java

```diff
@@ -0,0 +1,9 @@
+package org.code.protocol;
+
+public enum StatusMessageKey {
+  COMPILING,
+  COMPILATION_SUCCESSFUL,
+  RUNNING,
+  GENERATING_RESULTS,
+  EXITED
```

Comment:
nit: would Done/Finished be clearer? Or do we specifically want Exited here?

Reply:
I don't have a super strong opinion here, on the frontend we are displaying "completed" generally (what we do with this signal depends on the miniapp ). I can update this to "Completed" to make things match up.

Reply:
Ahh okay if it's different per miniapp then it doesn't really matter I suppose, I'm good with Exited too